.github/workflows/utility/validate_data.py

```
@@ -13,68 +13,73 @@ def checkChains():
-
-          if "logo_URIs" in chainSchema:
-            if "png" in chainSchema["logo_URIs"]:
-              validateRawGithubContent(chainSchema["logo_URIs"]["png"], True)
-            if "svg" in chainSchema["logo_URIs"]:
-              validateRawGithubContent(chainSchema["logo_URIs"]["svg"], True)
+                raise Exception("assetlist schema doesn't contain 'assets' array")
+
+          profilejson = os.path.join(networkfolder, chainfolder, "profilelist.json")
+          print(assetlistjson + "  - " + str(os.path.exists(profilejson)))
+          if os.path.exists(profilejson):
+            profileSchema = json.load(open(os.path.join(rootdir, profilejson)))
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)